spec/units/services/notification_handler_service_spec.rb

```diff
@@ -44,11 +44,15 @@
         .to change { ContentChange.count }.by(1)
     end
 
+    it "creates a MatchedContentChange" do
+      expect { described_class.call(params: params) }
+        .to change { MatchedContentChange.count }.by(1)
+    end
+
     it "enqueues the content change to be processed by the subscription content worker" do
-      allow(ContentChange).to receive(:create!).and_return(double(id: 1))
       expect(SubscriptionContentWorker)
         .to receive(:perform_async)
-        .with(1)
+        .with(kind_of(Integer))
```

Comment:
I think there's value in having the explicit value in here to prove we're actually passing the content change id and not something else by mistake. We do have higher level test coverage of this though...

Reply:
Agreed, I thought I needed this, but I've fixed it now so it's similar to how it was.